src/prepack-node.js

```diff
@@ -17,12 +17,11 @@ import { InitializationError } from "./prepack-standalone";
 import { prepackNodeCLI, prepackNodeCLISync } from "./prepack-node-environment";
 
 import type { Options } from "./options";
-import { defaultOptions } from "./options";
 
 export * from "./prepack-standalone";
 export * from "./prepack-node-environment";
 
-export function prepackStdin(options: Options = defaultOptions, callback: Function) {
+export function prepackStdin(options: Options, callback: Function) {
```

Comment:
Why are these options more important than any other option though? I just don't see the scenario where someone would want to learn about the intricate details of these optimizations. If they didn't want the global execution scope, they wouldn't have used Prepack to begin with and they would always want constant folding since its added perf cost is likely small comparatively.

Configuration hell has been a huge problem in the JS community recently, even coining phrases like "JS Fatigue", so I'm very sensitive to it. With things like Babel having individual plugins leading to presets instead and Webpack configuration being something that nobody really understands anymore. It's better to have good decisions made for you.